### ISSUE TITLE
LC0003 / LC0005 - False Positive - OptionMembers

### ISSUE THREAD

**Issue by pri-kise** [StefanMaron/BusinessCentral.LinterCop#199]
We receive the following warnings, that are clearly a false positive.

> warning LC0005: Wrong casing detected! Use  ,Hide Amounts,Hide Line instead.

> warning LC0003: Do not use an Object ID for properties or variables declaration. Use  ,Hide Amounts,Hide Line instead.

```al
field(50000; "My Option"; Option)
{
    Caption = 'My Option';
    DataClassification = CustomerContent;
    OptionCaption = ' ,Hide Amounts,Hide Line';
    OptionMembers = " ","Hide Amounts","Hide Line";
}
```

**Comment by Arthurvdv** [comment]
I'm quite sure this issue is resolved in the meanwhile. If you are still experiencing this issue, feel free to reopen this again.
